README.md

```diff
@@ -62,24 +64,27 @@ Accepted params:
 ```php
 require __DIR__ . '/vendor/autoload.php';
 
-use Auth0\SDK\API\Authentication;
+use Auth0\SDK\Auth0;
```

Comment:
Basically, I want to get rid of the oauth client dependency (mostly because the authentication class takes care of everything we need to build it).